packages/now-cli/src/util/output/create-output.ts

```diff
@@ -64,15 +65,28 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
 
   function error(
     str: string,
-    slug: string | null = null,
-    link: string | null = null,
+    slug?: string,
+    link?: string,
     action = 'More details'
   ) {
     print(`${chalk.red(`Error!`)} ${str}\n`);
     const details = slug ? `https://err.sh/now/${slug}` : link;
     if (details) {
-      print(`${action}: ${details}\n`);
+      print(`${chalk.bold(action)}: ${renderLink(details)}\n`);
+    }
+  }
+
+  function prettyError(err: Error & { link?: string; action?: string }) {
+    let str = err.message;
+    let link: string | undefined = undefined;
+    let action: string | undefined = undefined;
+    if (err.link) {
+      link = err.link;
+    }
+    if (err.action) {
+      action = err.action;
     }
```

Comment:
Are these two `if` statements necessary?

Seems like you could just `return error(str, undefined, err.link, err.action);` 🤔

Reply:
Good call. 85cb7ed